Kernel/Assertions.h

```diff
@@ -23,8 +25,13 @@ extern "C" {
 [[noreturn]] void abort();
 }
 
-#define VERIFY_INTERRUPTS_DISABLED() VERIFY(!(cpu_flags() & 0x200))
-#define VERIFY_INTERRUPTS_ENABLED() VERIFY(cpu_flags() & 0x200)
+#if ARCH(I386) || ARCH(X86_64)
+#    define VERIFY_INTERRUPTS_DISABLED() VERIFY(!(cpu_flags() & 0x200))
+#    define VERIFY_INTERRUPTS_ENABLED() VERIFY(cpu_flags() & 0x200)
+#else
+#    define VERIFY_INTERRUPTS_DISABLED() 
+#    define VERIFY_INTERRUPTS_ENABLED() 
```

Comment:
Can we FIXME this so we know we need to add it later?

Reply:
Added TODO()